docs/code/experimental/MlNetCookBook_StaticApi.md

```diff
@@ -0,0 +1,1026 @@
+var predictionFunc = model.CreatePredictionEngine<IrisInput, IrisPrediction>(mlContext);
+
+// Obtain the prediction. Remember that 'Predict' is not reentrant. If you want to use multiple threads
+// for simultaneous prediction, make sure each thread is using its own PredictionEngine.
+var prediction = predictionFunc.Predict(new IrisInput
+{
+    SepalLength = 4.1f,
+    SepalWidth = 0.1f,
+    PetalLength = 3.2f,
+    PetalWidth = 1.4f
+});
+```
+
+## What if my training data is not in a text file?
+
+The commonly demonstrated use case for ML.NET is when the training data resides somewhere on disk, and we use the `TextLoader` to read it.
+However, in real-time training scenarios the training data can be elsewhere: in a bunch of SQL tables, extracted from log files, or even generated on the fly.
+
+Here is how we can use [schema comprehension](SchemaComprehension.md) to bring an existing C# `IEnumerable` into ML.NET as a data view.
+
+For the purpose of this example, we will assume that we build the customer churn prediction model, and we can extract the following features from our production system:
+- Customer ID (ignored by the model)
+- Whether the customer has churned (the target 'label')
+- The 'demographic category' (one string, like 'young adult' etc.)
+- The number of visits from the last 5 days.
+```csharp
+private class CustomerChurnInfo
+{
+    public string CustomerID { get; set; }
+    public bool HasChurned { get; set; }
+    public string DemographicCategory { get; set; }
+    // Visits during last 5 days, latest to newest.
+    [VectorType(5)]
+    public float[] LastVisits { get; set; }
+}
+```
+
+Given this information, here's how we turn this data into the ML.NET data view and train on it:
+```csharp
+// Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+// as a catalog of available operations and as the source of randomness.
+var mlContext = new MLContext();
+
+// Step one: read the data as an IDataView.
+// Let's assume that 'GetChurnData()' fetches and returns the training data from somewhere.
+IEnumerable<CustomerChurnInfo> churnData = GetChurnInfo();
+
+// Turn the data into the ML.NET data view.
+// We can use CreateDataView or CreateStreamingDataView, depending on whether 'churnData' is an IList, 
+// or merely an IEnumerable.
+var trainData = mlContext.CreateStreamingDataView(churnData);
+
+// Build the learning pipeline. 
+// In our case, we will one-hot encode the demographic category, and concatenate that with the number of visits.
+// We apply our FastTree binary classifier to predict the 'HasChurned' label.
+
+// First, transition to the statically-typed data view.
+var staticData = trainData.AssertStatic(mlContext, c => (
+        HasChurned: c.Bool.Scalar,
+        DemographicCategory: c.Text.Scalar,
+        LastVisits: c.R4.Vector));
+
+// Build the pipeline, same as the one above.
+var staticLearningPipeline = staticData.MakeNewEstimator()
```

Comment:
>staticLearningPipeline [](start = 4, length = 22)

let's be consistent and pick one throughout the md: staticLearningPipeline or staticPipeline. I suggest just prefixing the name you chose in the dynamic.md file with 'static', so that when users copy-paste they don't end up mixing the default (dynamic) examples with this one.